app/views/idv/in_person/state_id/show.html.erb

```diff
@@ -145,7 +146,7 @@
       <% end %>
     <% end %>
     <%= render ValidatedFieldComponent.new(
-          name: :state_id_number,
+          name: :id_number, # Renaming form as a workaround for aggressive browser autofill assumptions where the word is an issue
```

Comment:
```suggestion
          name: :id_number, # Renaming form as a workaround for aggressive browser autofill assumptions
```

I just realized that, with "state" being removed form the name, people won't know what "the word" refers to, so maybe we can just omit this?

Reply:
Ope, that was supposed to say "the word state" - I'll fix it, thanks!